### ISSUE TITLE
gta 5 issue with vr 

### ISSUE THREAD

**Issue by Coyotewall** [LukeRoss00/gta5-real-mod#362]
I have a few issue, each time I try to tweak a setting my vr freezes and have to restart, when I get into gameplay I'm not in full vr but at a monitor shaped box, I do load into it and I do know its working but its never full vr, what resolution am I supposed to play at am I supposed to tweak settings in Nvidia control panel?
I did indeed follow the steps correctly It just doesn't give me full vr, 

I am using new script hook V with v7 mod, I also have followed different YouTube videos and I have got it to get inside the game but when I do go into gameplay there is a black  screen and I'm moving on my monitor screen but not on my headset
i have a medium end system so I chose the medium setting, 

some of my pc specs are 
intel i7 7700hq
Gtx 1060 6gb 
16 gb of ram
I am running on a laptop and that could  be the reason 
I'm using an oculus quest does it not work on it?
I followed the oculus quick steps and I have searched on the web for hours
I do not change the graphics settings manually but fx setting don't work good and also VSync  so with the low settings present you made it turns them off, 
The batch file doesn't report anything as far as I see the only thing I'm running into is my headset just shows a black screen and only sound and I (can) go into the menu.
Im not using any other mods just your vr mods.
I am using oculus vr, is there maybe a setting that makes it go into full vr?

**Comment by LukeRoss00** [comment]
Please see #239 when opening new issues

**Comment by Coyotewall** [comment]
I am using new script hook V with v7 mod, I also have followed different YouTube videos and I have got it to get inside the game but when I do go into gameplay there is a black screen and I'm moving on my monitor screen but not on my headset
i have a medium end system so I chose the medium setting,

some of my pc specs are
intel i7 7700hq
Gtx 1060 6gb
16 gb of ram
I am running on a laptop and that could be the reason
I'm using an oculus quest does it not work on it?
I followed the oculus quick steps and I have searched on the web for hours
I do not change the graphics settings manually but fx setting don't work good and also VSync so with the low settings present you made it turns them off,
The batch file doesn't report anything as far as I see the only thing I'm running into is my headset just shows a black screen and only sound and I (can) go into the menu.
Im not using any other mods just your vr mods.
I am using oculus vr, is there maybe a setting that makes it go into full vr?
